<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmbulanceI</class>
 <widget class="QDialog" name="AmbulanceI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>311</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton #pushButton{
	
	background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton #pushButton:hover{
	
	background-color: rgb(98, 185, 203);
}
QPushButton #pushButtonp:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(98, 185, 203);
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(27, 75, 101);
</string>
    </property>
    <property name="text">
     <string>Formulaire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>Matricule</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>Prix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>Date d'entretien</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>Etat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_mat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_prix">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajout">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}

</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modif">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_etat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="le_date">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	border-image: url(:/2A3/qt/tsawer/calendrier.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>1051</width>
     <height>341</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(27, 75, 101);</string>
    </property>
    <property name="text">
     <string>Liste des Ambulances</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_pfdA">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	border-image: url(:/2A3/qt/tsawer/pdf (2).png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_excel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	border-image: url(:/2A3/qt/tsawer/excel.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tabamb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="stat">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>statistique</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	qproperty-icon: url(:/Downloads/trier.png);
	qproperty-iconSize: 30px 30 px;
text-align:center;
	color: rgb(27, 75, 101);
}</string>
   </property>
   <property name="text">
    <string>Buzzer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>390</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	qproperty-icon: url(:/Downloads/trier.png);
	qproperty-iconSize: 30px 30 px;
text-align:center;
	color: rgb(27, 75, 101);
}</string>
   </property>
   <property name="text">
    <string>Etat</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pb_trierA">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Trier</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_tri_ambu">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>all</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matricule</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prix</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Etat</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	
	qproperty-icon: url(:/Downloads/trier.png);
	qproperty-iconSize: 30px 30 px;
text-align:center;
	color: rgb(27, 75, 101);
}</string>
    </property>
    <property name="text">
     <string>Trier par</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	
	qproperty-icon: url(:/Downloads/trier.png);
	qproperty-iconSize: 30px 30 px;
text-align:center;
	color: rgb(27, 75, 101);
}</string>
    </property>
    <property name="text">
     <string>RECHERCHE</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_ONA">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>360</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_OFFA">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:5px;
   border-right:1px solid #aaaaaa;
	border-bottom:1px solid #aaaaaa;
}
QPushButton:enabled{
background-color: rgb(98, 185, 203);
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(190, 233, 231);
	color: rgb(255, 255, 255);
	border-radius:5px;
}
QPushButton:disabled{
background-color: #aaaaaa;
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QTableView" name="tab_gazA">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>340</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="etatardA">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>390</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_suppamb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_supp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-image: url(:/2A3/qt/tsawer/poubella.png);
color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(27, 75, 101);</string>
   </property>
   <property name="text">
    <string>Matricule</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
